src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj

```diff
@@ -12,6 +12,20 @@
 		<SupportedOSPlatformVersion Condition="'$(TargetFramework)' == 'net6.0-ios'">14.2</SupportedOSPlatformVersion>
 		<NoWarn>Uno0001</NoWarn>
 	</PropertyGroup>
+	<!-- Linker configuration. -->
```

Comment:
Did you test your changes with an AppCenter build?

Reply:
Yes and it's also on our current projects migration.